Library/Homebrew/os/mac/pkgconfig/10.14/libunwind.pc

```diff
@@ -0,0 +1,11 @@
+prefix=/usr
+exec_prefix=/usr
+libdir=${exec_prefix}/lib/system
+includedir=${prefix}/include
```

Comment:
I did notice #5068, but it doesn't affect zeromq in my testing, since `libunwind` is a private requirement.

Without this pull request, it fails hard because it can't find `libunwind.pc`:

~~~
$ pkg-config --cflags libzmq
Package libunwind was not found in the pkg-config search path.
Perhaps you should add the directory containing `libunwind.pc'
to the PKG_CONFIG_PATH environment variable
Package 'libunwind', required by 'libzmq', not found
~~~

With this pull request, it succeeds, but I can't manage to see any reference to `libunwind` or `/usr/include` in the pkg-config outputs:

~~~
$ pkg-config --cflags libzmq
-I/usr/local/Cellar/zeromq/4.3.1/include
$ pkg-config --libs libzmq
-L/usr/local/Cellar/zeromq/4.3.1/lib -lzmq
$ pkg-config --static libzmq

~~~

I was hoping that #5068 wouldn't block this merge because things that depend on `zeromq` are broken right now. In parallel, I can take a look at helping with #5068.

Reply:
Ok, in the meantime I've submitted https://github.com/Homebrew/homebrew-core/pull/36121 to disable `libunwind` from the zeromq formula to unbreak its `pkg-config` file.